Resources/Textures/Structures/Machines/telecomms.rsi/meta.json

```diff
@@ -0,0 +1,34 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/a7df6036f04f386ac660bcc84c332a0b2ff5ed1a",
```

Comment:
i wont require you to but it would be helpful to link the dmi file you took this from at that commit instead